target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -28,7 +28,6 @@ ipq40xx_setup_interfaces()
 	dlink,dap-2610 |\
 	engenius,eap1300|\
 	engenius,emd1|\
-	extreme-networks,ws-ap3915i|\
```

Comment:
not part of this commit?

Reply:
sorry i've updated my local patch using new file, it's fixed now